socorro/processor/signature_utilities.py

```diff
@@ -93,6 +93,63 @@ def __init__(self, config, quit_check_callback=None):
         self.fixup_comma = re.compile(r',(?! )')
 
     #--------------------------------------------------------------------------
+    def _is_exception(
+        self,
+        exception_list,
+        remaining_original_line,
+        line_up_to_current_position
+    ):
```

Comment:
Why is this a class method?
Seems it's only used from within the loop in `_collapse`